.github/update_c2pa.yml

```diff
@@ -0,0 +1,29 @@
+name: Update c2pa_rs Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      c2pa_version:
+        description: 'c2pa-rs version (e.g., 0.51.1)'
+        required: true
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Update C2PA_VERSION in CMakeLists.txt
+      run: |
+        version="${{ github.event.inputs.c2pa_version }}"
+        sed -i "s/set(C2PA_VERSION \".*\")/set(C2PA_VERSION \"${version}\")/" CMakeLists.txt
+
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v6
```

Comment:
https://github.com/peter-evans/create-pull-request Not sure it's an officially anointed one, but I think we can use it.